libfeed/feed-base-card-store.c

```diff
@@ -16,6 +16,7 @@
  */
 
 #include "feed-base-card-store.h"
+#include "feed-enums.h"
```

Comment:
I appreciate that doing this in a new commit is a little awkward, but I had to do it this way since I needed something to generate the enum types from and didn't want to include that in the same commit as the one that introduced the BaseCardStore